src/FamilyService.Validation/Child/EditChildRequestValidator.cs

```diff
@@ -0,0 +1,76 @@
+using System;
+using System.Text.RegularExpressions;
+using System.Collections.Generic;
+using FluentValidation;
+using Microsoft.AspNetCore.JsonPatch;
+using Microsoft.AspNetCore.JsonPatch.Operations;
+using LT.DigitalOffice.Kernel.Validators;
+using LT.DigitalOffice.FamilyService.Models.Dto.Enums;
+using LT.DigitalOffice.FamilyService.Models.Dto.Requests.Child;
+using LT.DigitalOffice.FamilyService.Validation.Child.Interfaces;
+
+namespace LT.DigitalOffice.FamilyService.Validation.Child
+{
+  public class EditChildRequestValidator : BaseEditRequestValidator<EditChildRequest>, IEditChildRequestValidator
+  {
+    private static Regex NameRegex = new(@"^([a-zA-Zа-яА-ЯёЁ]+|[a-zA-Zа-яА-ЯёЁ]+[-\s`]?[a-zA-Zа-яА-ЯёЁ]+|[a-zA-Zа-яА-ЯёЁ]+[-\s`]?[a-zA-Zа-яА-ЯёЁ]+[-\s`]?[a-zA-Zа-яА-ЯёЁ]+)$");
+    
+    private void HandleInternalPropertyValidation(
+      Operation<EditChildRequest> requestedOperation,
+      ValidationContext<JsonPatchDocument<EditChildRequest>> context)
+    {
+      RequestedOperation = requestedOperation;
+      Context = context;
+
+      AddСorrectPaths(
+        new List<string> {
+          nameof(EditChildRequest.Name),
+          nameof(EditChildRequest.Gender),
+          nameof(EditChildRequest.IsActive),
+          nameof(EditChildRequest.Info)
+        });
+
+      AddСorrectOperations(nameof(EditChildRequest.Name), new List<OperationType> { OperationType.Replace });
+      AddСorrectOperations(nameof(EditChildRequest.Gender), new List<OperationType> { OperationType.Replace });
+      AddСorrectOperations(nameof(EditChildRequest.IsActive), new List<OperationType> { OperationType.Replace });
+      AddСorrectOperations(nameof(EditChildRequest.Info), new List<OperationType> { OperationType.Replace });
+      
+      AddFailureForPropertyIf(
+        nameof(EditChildRequest.Name),
+        o => o == OperationType.Replace,
+        new Dictionary<Func<Operation<EditChildRequest>, bool>, string> {
+          { x=> !string.IsNullOrEmpty(x.value?.ToString().Trim()), "Child's name mustn't be empty."},
+          { x => NameRegex.IsMatch(x.value.ToString().Trim()), "Name contains invalid characters." },
+          { x => x.value.ToString().Trim().Length <= 45, "Maximum name lenght is 45."}
```

Comment:
better <46

Reply:
Don't close the conversations, 'till you make changes!

Reply:
Fix it.

Reply:
We have discussed it with @vorobevamv and came to conclusion to leave it at my discretion.